src/main/java/uk/gov/hmcts/ccd/domain/service/common/AuthorisedCaseDataFilter.java

```diff
@@ -0,0 +1,10 @@
+package uk.gov.hmcts.ccd.domain.service.common;
```

Comment:
can we move all the filters and post processors to a `security` or similar subpackage?

Reply:
moved only couple of classes, if I move all access related classes, it affects 120 files. So this could be done as part of a separate tech debt.

Reply:
also see comment below on the implementation class

Reply:
I meant to move only the new security classes we are creating in the new search api. Not the whole security classes

Reply:
moved new classes to security package